tests/Feature/PostSlugModelTest.php

```
@@ -0,0 +1,151 @@
+    public function testExplicitSlugSurvivesInsert(): void
+    {
+        $posts = $this->posts();
+        $posts->insert([
+            'title' => '멀티보드 만들기 3 - 스키마 설계',
+            'body'  => '본문',
+            'slug'  => 'ci4-multiboard-03-schema-blueprint',
+        ]);
+
+        $this->assertSame(
+            'ci4-multiboard-03-schema-blueprint',
+            $posts->find($posts->getInsertID())->slug
```

PHPStan (2.2.7)

[error] Cannot access property $slug on array<int|string, bool|float|int|object|string|null>|object.


(property.nonObject)